tasks/www.rb

```diff
@@ -0,0 +1,90 @@
+# encoding: utf-8
+# Copyright:: Copyright (c) 2015 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require 'inquirer'
+
+namespace :www do
+  desc 'Builds the site locally'
+  task :build do
+    system('cd www/ && middleman build')
+  end
+
+  desc 'Releases the site to gh-pages'
+  task :release do
+    # This folder contains the built files
+    dst = 'www/build'
+    unless File.directory?(dst) && File.file?(File.join(dst, 'index.html'))
+      puts 'It looks like you have not built the site yet. Calling rake www:build'
+      Rake::Task['www:build'].invoke
+    end
+
+    unless File.directory?(dst) && File.file?(File.join(dst, 'index.html'))
+      fail 'It looks like the siste was not build. Aborting.'
+    end
+
+    # check if git exists
+    system('command -v git >/dev/null 2>&1') ||
+      fail("It looks like `git` isn't installed. It is required to run this build task.")
+
+    unless system('git diff-index --quiet HEAD --')
+      fail 'Please make sure you have no uncommitted changes in this repository.'
+    end
+
+    file_count = Dir['www/build/*'].length
+    file_size = `du -hs www/build`.sub(/\s+.*$/m, '')
+
+    puts '----> Remove local gh-pages branch'
+    system('git branch -D gh-pages')
```

Comment:
That's a great idea! Let's add a mechanism after this MR

added to https://github.com/chef/inspec/issues/1114 for tracking